### DISCUSSION TITLE
Cant set the target temperature on an HM-CC-RT-DN via script

### DISCUSSION THREAD

**Discussion by heli77** [SukramJ/aiohomematic#715] [Q&A]
Hey folks,
sorry but I think i am just too stupid but I cant find a way to set the target temperature on an HM-CC-RT-DN in an script. Yes I can set it via an climate card. I tried to use the temperature entity in Yaml as described in HVAC Documents but I get an error. 
How can I set the temperature then?

kind regards
Heli

**Comment by SukramJ** [comment]
This is not specific to this integration and  can be found in the [HA docs](https://www.home-assistant.io/integrations/climate/#service-climateset_temperature).
